pkgs/by-name/to/touche-driver/package.nix

```
@@ -0,0 +1,38 @@
+{
+  rustPlatform,
+  lib,
+  fetchFromGitHub,
+  pkg-config,
+  systemd,
+}:
+rustPlatform.buildRustPackage rec {
```

would be better to use finalAttrs here to make it easier to override
```suggestion
rustPlatform.buildRustPackage (finalAttrs: {
```